i-slide.js

```diff
@@ -155,9 +156,14 @@ class ISlide extends HTMLElement {
       this.setAttribute('type', value);
     }
 
+    // Start loading PDF scripts if needed
+    if (this.#type === 'application/pdf') {
+      this.#loadPDFScripts();
+    }
+
     // Trigger a fetch-and-render cycle on next tick if value changed, unless
     // that's already planned
-    if ((this.#src !== oldValue) && !this.#renderCyclePlanned) {
+    if ((this.#type !== oldValue) && !this.#renderCyclePlanned) {
```

Comment:
is that really needed now that both fetch and render are independent of the type attribute?

Reply:
Probably not for now. Now, if the app sets the `type` attribute to `text/html` whereas the current slideset is a PDF document, we should probably report an error and maybe not render the slide?

Reply:
if we are to mimic the [`type` attribute of the `object` element](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-object-type), and if I understand [the algorithm](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-object-element:dom-manipulation-task-source) correctly, `type` is a mostly a hint, not something we should fail over